src/components/Calculator.js

```diff
@@ -1,53 +1,46 @@
-        <button type="button" onClick={this.buttonPressHandler} className="state-number">5</button>
-        <button type="button" onClick={this.buttonPressHandler} className="state-number">6</button>
-        <button type="button" onClick={this.buttonPressHandler} className="action-btn">-</button>
-        <button type="button" onClick={this.buttonPressHandler} className="state-number">1</button>
-        <button type="button" onClick={this.buttonPressHandler} className="state-number">2</button>
-        <button type="button" onClick={this.buttonPressHandler} className="state-number">3</button>
-        <button type="button" onClick={this.buttonPressHandler} className="action-btn">+</button>
-        <button type="button" onClick={this.buttonPressHandler} className="state-number mod">0</button>
-        <button type="button" onClick={this.buttonPressHandler} className="state-number">.</button>
-        <button type="button" onClick={this.buttonPressHandler} className="action-btn">=</button>
+  return (
+    <div className="container">
+      <div type="number" className="field-inp">
+        {total}
+        {operation}
+        {next}
       </div>
-    );
-  }
-}
+      <button type="button" onClick={buttonPressHandler} className="state-number">AC</button>
+      <button type="button" onClick={buttonPressHandler} className="state-number">+/-</button>
+      <button type="button" onClick={buttonPressHandler} className="state-number">%</button>
+      <button type="button" onClick={buttonPressHandler} className="action-btn">÷</button>
+      <button type="button" onClick={buttonPressHandler} className="state-number">7</button>
+      <button type="button" onClick={buttonPressHandler} className="state-number">8</button>
+      <button type="button" onClick={buttonPressHandler} className="state-number">9</button>
+      <button type="button" onClick={buttonPressHandler} className="action-btn">x</button>
+      <button type="button" onClick={buttonPressHandler} className="state-number">4</button>
+      <button type="button" onClick={buttonPressHandler} className="state-number">5</button>
+      <button type="button" onClick={buttonPressHandler} className="state-number">6</button>
+      <button type="button" onClick={buttonPressHandler} className="action-btn">-</button>
+      <button type="button" onClick={buttonPressHandler} className="state-number">1</button>
+      <button type="button" onClick={buttonPressHandler} className="state-number">2</button>
+      <button type="button" onClick={buttonPressHandler} className="state-number">3</button>
+      <button type="button" onClick={buttonPressHandler} className="action-btn">+</button>
+      <button type="button" onClick={buttonPressHandler} className="state-number mod">0</button>
+      <button type="button" onClick={buttonPressHandler} className="state-number">.</button>
+      <button type="button" onClick={buttonPressHandler} className="action-btn">=</button>
+    </div>
+  );
+};
```

Comment:
- [OPTIONAL] When using the modulus (%) with 0 it crashes the app. I think this should work in the same manner as dividing by 0 does. ♻️ 

![mod%0](https://user-images.githubusercontent.com/79876959/143510767-86c3f755-d949-4ad9-b93a-5181006ddcd2.PNG)